packages/scene-composer/src/components/three-fiber/Environment.tsx

```diff
@@ -7,6 +7,7 @@ import DirectionalHDR from '../../assets/hdri/Directional_sm.hdr';
 import ChromaticHDR from '../../assets/hdri/Chromatic_sm.hdr';
 
 export const presets = {
+  noPreset: '' as string,
```

Comment:
Not blocking and I see the pattern below of this, but I believe there should be a way to achieve the same with casting.

Reply:
I'll take a look & see what I can do here. :)

Reply:
Fixed.